src/plugins/intel_gpu/tests/fusions/convolution_fusion_test.cpp

```diff
@@ -108,20 +129,21 @@ class ConvFusingTest : public BaseFusingTest<convolution_test_params> {
             std::cout << "kernel: " << info_fused->kernel_id << std::endl;
     }
 
-    layout get_input_layout(convolution_test_params& p) {
+    layout get_input_layout(T_param& p) {
         auto pad = p.pad;
         std::vector<int> pad_ = { 0, 0, static_cast<int>(pad[1]), static_cast<int>(pad[0]) };
         return layout{ p.data_type, p.input_format, p.in_shape, padding{ pad_ } };
     }
 
-    layout get_per_channel_layout(convolution_test_params& p) {
+    layout get_per_channel_layout(T_param& p) {
         return layout{ p.default_type, p.default_format, tensor{1, p.out_shape.feature[0], 1, 1} };
     }
 
-    layout get_prelu_slope_layout(convolution_test_params& p) {
+    layout get_prelu_slope_layout(T_param& p) {
         return layout{ p.default_type, p.input_format, tensor{1, p.out_shape.feature[0], p.out_shape.spatial[0], 1} };
     }
 };
+using ConvFusingTest=ConvFusingTestBase<convolution_test_params>;
```

Comment:
I'm afraid it makes debugging complicated. We already have several implementation of BaseFusingTest with different argument. What about just adding another implementation?